docs/benchmarks.md

```diff
@@ -19,8 +19,8 @@
 
 ### GCSFuse command
 ```
-gcsfuse --implicit-dirs --stat-cache-ttl=60s --type-cache-ttl=60s
- --disable-http2 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
+gcsfuse --implicit-dirs --stat-cache-ttl=60s --type-cache-ttl=60s 
+--client-protocol http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
```

Comment:
use client-protocol=http1 to keep in consistent with other flags

Reply:
changed.